integr8ly/integreatly-operator
# INTLY-2830 / INTLY-4959 - bump solution explorer operator version

## Description
Bump the solution explorer operator version to use operator with the image with the logout for OS4.x fix and login crash fix.
Should be merged after the following PRs have been merged:

* Logout Fix
  * https://github.com/integr8ly/tutorial-web-app/pull/544
  * https://github.com/integr8ly/tutorial-web-app-operator/pull/83/files
* Login crash Fix
  * https://github.com/integr8ly/tutorial-web-app/pull/547
  * https://github.com/integr8ly/tutorial-web-app-operator/pull/85

Jira:
* https://issues.redhat.com/browse/INTLY-2830
* https://issues.redhat.com/browse/INTLY-4959

## Verification 
* When the above pull requests have been merged. Install integreatly from this branch
* Run the `./scripts/setup-htpass-idp.sh` to create sample users
* Visit solution explorer and ensure you can sign in and sign out without errors with the created users
